github/issues.go

```diff
@@ -21,8 +21,8 @@ type IssuesService service
 //
 // Note: As far as the GitHub API is concerned, every pull request is an issue,
 // but not every issue is a pull request. Some endpoints, events, and webhooks
-// may also return pull requests via this struct. If PullRequestLinks is nil,
-// this is an issue, and if PullRequestLinks is not nil, this is a pull request.
+// may also return pull requests via this struct. The issue is a pull request
+// if IsPullRequest returns true.
```

Comment:
Let's not change this. Revert the doc change, but add a sentence that follows:

```
// The IsPullRequest helper makes it easier to check that.
```

Basically, I don't think we want to make existing users of `go-github` have to change their existing code. They don't. They should only use this helper if they wish, if it's helpful, and feel okay with not using it.